### PR TITLE
removing trvapply-vrtdemande domain

### PR DESCRIPTION
**Pull request by ben851**
# Summary | Résumé

The trvapply-vrtdemande domain is no longer registered and we got clearance from the service owners to remove this custom sending domain.

I'm also adding in an auto tag for [review] on git hooks because I keep forgetting.

## Related Issues | Cartes liées

* https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/448

## Before merging this PR

Read code suggestions left by the
[cds-ai-codereviewer](https://github.com/cds-snc/cds-ai-codereviewer/) bot. Address
valid suggestions and shortly write down reasons to not address others. To help
with the classification of the comments, please use these reactions on each of the
comments made by the AI review:

| Classification      | Reaction | Emoticon |
|---------------------|----------|----------|
| Useful              | +1       | 👍        |
| Noisy               | eyes     | 👀        |
| Hallucination       | confused | 😕        |
| Wrong but teachable | rocket   | 🚀        |
| Wrong and incorrect | -1       | 👎        |

The classifications will be extracted and summarized into an analysis of how helpful
or not the AI code review really is.

## Test instructions | Instructions pour tester la modification

TF Apply works
Verify that the domain is no longer in prod.

## Release Instructions | Instructions pour le déploiement

None.

## Reviewer checklist | Liste de vérification du réviseur

* [x] This PR does not break existing functionality.
* [x] This PR does not violate GCNotify's privacy policies.
* [x] This PR does not raise new security concerns. Refer to our GC Notify Risk Register document on our Google drive.
* [x] This PR does not significantly alter performance.
* [x] Additional required documentation resulting of these changes is covered (such as the README, setup instructions, a related ADR or the technical documentation).

> ⚠ If boxes cannot be checked off before merging the PR, they should be moved to the "Release Instructions" section with appropriate steps required to verify before release. For example, changes to celery code may require tests on staging to verify that performance has not been affected.

### CODE REVIEW

File: aws/dns/ses.tf [hunk 1]
```diff
@@ -91,48 +81,8 @@ resource "aws_ses_active_receipt_rule_set" "main" {
 
 ###
 # Additional custom sending domains for emails
-# trvapply-vrtdemande.apps.cic.gc.ca is alone for historic reasons
-# and is not refactored to make sure the ressource is not destroyed/recreated.
-# Read the section "Refactoring Can Be Tricky"
-# https://blog.gruntwork.io/terraform-tips-tricks-loops-if-statements-and-gotchas-f739bbae55f9
-#
-# Afterwards there is a more automated way, using the set variable
-# `ses_custom_sending_domains`.
```
Comment (jimleroyer) [inline aws/dns/ses.tf:100]: Well it is even nicer to get rid of this then.

File: hooks/commit-msg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+COMMIT_MSG=`head -n1 $ARGS`
```
Comment (jimleroyer) [inline hooks/commit-msg:1]: There should be a hashbang as first line.

**Review by jimleroyer — CHANGES_REQUESTED** [review]
Asking questions about why we are reverting how the review bot works.

File: hooks/commit-msg [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+PATTERN="\[review\]"
+if ! [[ "$COMMIT_MSG" =~ $PATTERN ]]; then
+  echo "Missing the review tag. Adding it in automatically."
+  echo "[review]" >> $1
```
[the remark this answers is not in the document]
Comment (ben851) [reply hooks/commit-msg]: Switched it to accept a prompt

File: hooks/commit-msg [hunk 4]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+COMMIT_MSG=`head -n1 $ARGS`
+PATTERN="\[review\]"
+# Allows us to read user input below, assigns stdin to keyboard
+exec < /dev/tty
+
+if ! [[ "$COMMIT_MSG" =~ $PATTERN ]]; then
+while true; do
+
+  read -p "[post-commit hook] Missing the review tag. Add it automatically? (Y/n) " yn
```
Comment (jimleroyer) [inline hooks/commit-msg:10]: Now that is freakin awesome.

**Review by jimleroyer — APPROVED** [review]
LGTM

**Pull request merged**